### PR TITLE
Add back traits, traitsui deps


### PR DESCRIPTION
**Pull request by hoechenberger**

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -78,9 +78,9 @@ outputs:
         - mffpy >=0.5.7
         - imageio >=2.6.1
         - imageio-ffmpeg >=0.4.1
+        - traits
+        - traitsui
```
Comment (larsoner) [inline recipe/meta.yaml:82]: Why? We shouldn't need these anymore. All we should need is `traitlets`. Maybe that should live in `requirements_base.txt` and in `mne-base`

Comment (hoechenberger) [reply recipe/meta.yaml:82]: I have to say I'm a bit confused re dependencies in setup.pay and what we install in requirements.txt, and even within that file there are some deps that seem kind of redundant. @christianbrodbeck had explicitly added traitlets in #61, so I thought we need them?

Comment (larsoner) [reply recipe/meta.yaml:82]: `traitlets`, yes, but not `traits` or `traitsui`. Historically the latter two were only needed for mayavi-based coreg and are now used only with mne-kit-gui

Comment (larsoner) [reply recipe/meta.yaml:82]: Can you open a PR to revert?

Comment (hoechenberger) [reply recipe/meta.yaml:82]: Yes will do!

**Pull request merged**